package.json

```diff
@@ -30,7 +31,7 @@
     "babel-preset-react": "^6.24.1",
     "express": "^4.16.1",
     "flow-bin": "^0.41.0",
-    "jest": "^21.2.1",
+    "jest": "23",
```

Comment:
I ran issues when I tried to update to `27` so I gave `23` a try and it work and has support for inline snapshots.